### PR TITLE
Sidekiq

### PR DESCRIPTION
**Pull request by iezer**
@mixonic  If you have a moment to look this over that would be much appreciated. This PR sets up an Async work called `CvRequestWorker` which sends a request to the Lion Guardian's CV server when creating a `CvRequest`. Also creates another worker called `CvResultsWorker` to check for results and keep checking every 10 seconds until results come back.

CC @redyaffle

### CODE REVIEW

**Comment by iezer** [comment]
merging.

File: README.md [hunk 1]
```diff
@@ -10,8 +10,10 @@ LG API (Rails)
 * Rubygems
 * Bundler
 * Postgres DB - `brew install git postgres` on Mac
+* Redis
 
 ## Setup
  * `bundle install`
  * `bundle exec rake db:setup`
  * `bundle exec rails server`
+ * `bundle exec sidekiq` in a separate terminal to run sidekiq which will run async jobs for communicating with CV Server
```
Comment (mixonic) [inline README.md:19]: I believe both:

```
bundle exec rails server
bundle exec sidekiq
```

Can be replaced with

```
gem install foreman # only once, obviously
foreman start
```

[Foreman](http://ddollar.github.io/foreman/) boots procfiles just like Heroku

File: app/workers/cv_results_worker.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class CvResultsWorker
+  include Sidekiq::Worker
+  require 'lg/cv_results_reader'
+
+  DELAY_IN_SECONDS = 10
```
Comment (mixonic) [inline app/workers/cv_results_worker.rb:5]: Is this a timeout? `TIMEOUT_IN_SECONDS`

File: config/routes.rb [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   devise_for :users, controllers: { sessions: 'sessions' }
 
   ActiveAdmin.routes(self)
 
+  admin_constraint = lambda { |request|
+    request.env["warden"].authenticate? and request.env['warden'].user.class == AdminUser
+  }
```
Comment (mixonic) [inline config/routes.rb:11]: :clap:

File: lib/lg/cv_results_reader.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
+require 'rest-client'
+
+module LG
+  class CvResultsReader
+
+    CV_SERVER_URL = 'https://simbadx.herokuapp.com'
```
Comment (mixonic) [inline lib/lg/cv_results_reader.rb:6]: Should likely be set per-environment and shared between this and the other lib

**Pull request merged**